index.html

```diff
@@ -13300,7 +13302,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-valuetext">
 			<pdef>aria-valuetext</pdef>
 			<div class="property-description">
-				<p><a>Defines</a> the human readable text alternative of <pref>aria-valuenow</pref> for a range <a>widget</a>.</p>
```

Comment:
I suppose this kind of makes it sound like it creates the human-readable version of aria-valuenow for combobox, but aria-valuenow doesn't work in a combobox.

Reply:
yeah.  you're right...  going to need to change this.  thanks